### PR TITLE
endorsement page

### LINKED ISSUE #867: Endorsement page
### Describe the feature

This page should display our animated ring component to give some sense of outreach. On the right side the visitor should have the ability to sign-up for our newsletter. I've created a "wireframe".

Frame 74

### Why is this feature needed?

The endorsement page should be our "outcall" page to give our supporters the ability to show the endorsement.

### Additional context

"Endorsement" ... support page would be in conflict with actual support requests. Open for suggestions.

_No response_

### PR DESCRIPTION
**Pull request by DasProffi**
## Which issues does this pull request close?
closes #867

### CODE REVIEW

**Comment by MaxSchaefer** [comment]
@DasProffi I like this page a lot! Already good on visual. What about the text?

**Comment by DasProffi** [comment]
> @DasProffi I like this page a lot! Already good on visual. What about the text?
> 

The text isn't defined yet, therefore I didn't add it, but I can add placeholders for now to see the design

**Comment by use-to** [comment]
Also, the mobile version is still a little bit buggy. We should also add a `press` option as a newsletter option.

File: landing-page/pages/newsletter/index.tsx [hunk 1]
```diff
@@ -38,12 +41,49 @@ const submitNewsLetterForm = (form: NewsLetterFormType) => submitHubSpotForm(
...
+            {width !== 0 && height !== 0 && (
+              <>
+                <RadialRings
+                  sizeCircle1={sizeCircle1}
+                  sizeCircle2={sizeCircle2}
+                  sizeCircle3={sizeCircle3}
+                  waveWidth={waveWidth}
+                />
+                <Span className={tw(`absolute z-[100] top-1/2 left-[${sizeCircle2}px] -translate-x-1/2 -translate-y-1/2 text-white mobile:hidden`)}>Goal 1</Span>
+                <Span className={tw(`absolute z-[100] top-1/2 left-[${sizeCircle2 + (sizeCircle3 - sizeCircle2) / 2}px] -translate-x-1/2 -translate-y-1/2 text-white mobile:hidden`)}>Goal 2</Span>
```
Comment (use-to) [inline landing-page/pages/newsletter/index.tsx:82]: Lets hide these goals for now.
```suggestion
                <Span className={tw(`absolute z-[100] top-1/2 left-[${sizeCircle2}px] -translate-x-1/2 -translate-y-1/2 text-white mobile:hidden`)}></Span>
                <Span className={tw(`absolute z-[100] top-1/2 left-[${sizeCircle2 + (sizeCircle3 - sizeCircle2) / 2}px] -translate-x-1/2 -translate-y-1/2 text-white mobile:hidden`)}></Span>
```

**Review by use-to — CHANGES_REQUESTED** [review]
Please apply the following changes:
- [x] Move/Rename this site to `/join`
- [x] Remove the mock goals (see below)
- [ ] Link this page on the landing page and rename the `Discover Our Vision` button to `Subscribe` (or similar)
- [ ] Remove the `Try our demo!` button

**Review by use-to — APPROVED** [review]
Good job 👏

**Pull request merged**